pandas/tests/series/test_analytics.py

```diff
@@ -220,7 +231,7 @@ def test_npdiff(self):
         s = Series(np.arange(5))
 
         r = np.diff(s)
-        assert_series_equal(Series([nan, 0, 0, 0, nan]), r)
+        assert_series_equal(Series([np.nan, 0, 0, 0, np.nan]), r)
```

Comment:
OK for now but typically best if you do things like this in separate PRs as its unrelated to the issue you are trying to solve and makes the review process more difficult